scout/constants/indexes.py

```diff
@@ -12,17 +12,14 @@
         IndexModel(
             [("build", ASCENDING), ("chromosome", ASCENDING)],
             name="build_chromosome",
-            background=True,
```

Comment:
👍🏻 Hope people using scout are also using a DB version >7 🤔

Reply:
Right, but probably best to stick with the times anyway? The "already existing" indices not backgrounding is not going to hurt, and a fresh install would hopefully be made on a current engine. The worst for a potential existing large old mongod install is the new index, but then it will be only that single one, and it would not be production ready immediately anyway.

Reply:
Agreed, let's merge!